src/utils/errorClasses.js

```diff
@@ -22,9 +22,15 @@ class ManifestNotFoundError extends TypedError {
     super(message || `"manifest.json" não encontrado para a extensão em "${manifestPath}", Execute "qt link-extension"`, 'manifest-not-found')
   }
 }
+class EntryPointNotFoundInPackageError extends TypedError {
+  constructor ({ message, entryPointPath } = {}) {
+    super(message || `O entrypoint especificado (${entryPointPath}) não está entre as extensões que já foram selecionadas. Tem certeza que o caminho está correto ou que a extensão já foi selecionada com qt select-extension?`)
```

Comment:
@luizAlbuquerqueJr, o comando agr é link-extension :eyes:.
(Eu vi q tu só copiou e colou pra documentar o erro)
```suggestion
    super(message || `O entrypoint especificado (${entryPointPath}) não está entre as extensões que já foram selecionadas. Tem certeza que o caminho está correto ou que a extensão já foi selecionada com qt link-extension?`)
```

Reply:
done